tests/leftNavigation_test.js

```diff
@@ -55,3 +55,29 @@ Scenario(
     assert.ok(tabsCount1 === tabsCount2, 'Settings page isn\'t opened in the same tab');
   },
 );
+
+Scenario(
+  'PMM-9550 Verify downloading server diagnostics logs',
```

Comment:
Add right tags, it should be `@settings` here

Reply:
hmm, could be `@menu` thats fine, just need some job to also execute this, menu seems fine.

Reply:
Done 👍  We already have tests with that tag, maybe there's a job already